include/fmt/format-inl.h

```diff
@@ -49,7 +49,7 @@ FMT_BEGIN_NAMESPACE
 namespace detail {
 
 FMT_FUNC void assert_fail(const char* file, int line, const char* message) {
-  print(stderr, "{}:{}: assertion failed: {}", file, line, message);
+  std::fprintf(stderr, "%s:%d: assertion failed: %s", file, line, message);
```

Comment:
Could you add a short comment explaining why we use printf here. It's too tempting to change it back =).

Reply:
Good idea, done.